examples/declarative-linking/pom.xml

```diff
@@ -50,7 +50,7 @@
         </dependency>
 
         <dependency>
-            <groupId>org.glassfish.web</groupId>
+            <groupId>org.glassfish</groupId>
```

Comment:
This should be updated in root pom, too. Anyway, since el was contributed to the EF, too, we need to have dependency on this new version of el, which has not been released, yet

Reply:
Sorry I missed the change the root pom...